elastic/kibana
# [Cases] Table Solution Filter Not Rendering Any Checked Option When All Selected

## Sumarry

Closes https://github.com/elastic/kibana/issues/172454

## Testing
- Verify that the solutions filter works correctly. Click on all solutions to verify that you can filter by all of them
- Verify that the solution filter is not showing if you have access only to one solution
- Verify that the solution filter shows only the solution you have access to (for example Security solution and Stack)
- Verify that the solution filter is not showing when you attach alerts from within the solutions and you see the correct cases
- Verify that you can see the correct cases on each solution when you have access to all solutions



Flaky test runner: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/4506
